runtime/lua/vim/loader.lua

```diff
@@ -141,7 +165,7 @@ end
 function Loader.loader(modname)
   local ret = M.find(modname)[1]
   if ret then
-    local chunk, err = Loader.load(ret.modpath, { hash = ret.stat })
+    local chunk, err = loadfile(ret.modpath)
```

Comment:
Wait, we're still ging to track this twice. Should be Loader.load here?

Reply:
The goal of this PR is to use `_G.loadfile`, whatever that may be. This way user can wrap and profile `loadfile` themselves whether the cache is on/off.

Reply:
ok, no big deal.